src/app/trainee/mock-data/trainee-spec-data.ts

```diff
@@ -0,0 +1,20 @@
+// NOTE only import into tests or mock. Must remove from all imports from  production scripts
```

Comment:
is it possible to move these responses to an external mocking api of some sort so this code doesn't get included in the build bundles? then once the BE api is ready it would just be a case of updating the environment.ts file

Reply:
spec-data and spec-components with "spec" only imported into test files. Will remove reference from state-management so no chance of being imported into builds.